lib/peeps.rb

```diff
@@ -0,0 +1,30 @@
+require 'pg'
+
+class Peep
+  def self.all
+    if ENV['ENVIRONMENT'] == 'test'
+      connection = PG.connect(dbname: 'chitter_test')
+    else
+      connection = PG.connect(dbname: 'chitter')
+    end  
+    result = connection.exec("SELECT * FROM peeps;")
+    result.map { |peep| peep['message'] }
+
+    @peeps = [
```

Comment:
You get the peeps from the database on lines 10 - 11 but then you don't use them because @peeps (on line 13) is hardcoded. You should use result instead and assign that to **@peeps** :)